acir/Cargo.toml

```
@@ -12,6 +12,7 @@ rust-version.workspace = true
 [dependencies]
 acir_field.workspace = true
 serde.workspace = true
+brillig_bytecode = { version = "0.1.0", path = "../brillig_bytecode" }
```

```suggestion
brillig_bytecode = { version = "0.1.0", path = "../brillig_bytecode", no-default-features = true }
```

This is needed once we set a default feature flag otherwise we'll only be able to compile with bn254.

For good measure we should update the feature flags in this file to include `brillig_bytecode/bn254`, etc.